beacon-chain/sync/querier.go

```diff
@@ -71,6 +92,27 @@ func (q *Querier) Stop() error {
 	return nil
 }
 
+func (q *Querier) listenForChainStart() {
+
+	chainStartChan := make(chan time.Time, 1)
+
+	sub := q.powchain.ChainStartFeed().Subscribe(chainStartChan)
+	defer sub.Unsubscribe()
+	for {
+		select {
+		case <-chainStartChan:
+			q.isChainStart = true
+			return
+		case <-sub.Err():
+			log.Debug("Subscriber closed, exiting goroutine")
```

Comment:
This is super important - should we be fatal'ing if this fails? Otherwise the entire rest of sync breaks down